<launch>
	<!-- Trajectory Generation Specific Parameters -->
	<arg name="samplingTipMargin" 		default="4"  /> 
	<arg name="samplingOutMargin" 		default="16" /> 
	<arg name="samplingSpeedFactor" 	default="0.25" />		
	<arg name="enableHeadingSmoothing" 	default="false" />
			
	<node pkg="local_planner" type="rollout_generator_node" name="rollout_generator_node" output="screen">
		<param name="samplingTipMargin" 		value="$(arg samplingTipMargin)"  /> 
		<param name="samplingOutMargin" 		value="$(arg samplingOutMargin)" /> 
		<param name="samplingSpeedFactor" 		value="$(arg samplingSpeedFactor)" />		
		<param name="enableHeadingSmoothing" 	value="$(arg enableHeadingSmoothing)" />	
	</node>				
			
</launch>
